src/net.h

```diff
@@ -10,6 +10,7 @@
 #include "compat.h"
 #include "limitedmap.h"
 #include "netbase.h"
+#include "primitives/block.h"
```

Comment:
Please remove `primitives/block.h` from the network layer. Compute the size of the block in `main.cpp` and pass the size instead of the block itself into `FilterStatsProcessBlock()`.